### PR TITLE
[OCRVS-6437] Forward Elastalert emails through country config

### PR DESCRIPTION
**Pull request by rikukissa**
In this new model, instead of Elastalert sending emails directly using SMTP details, it sends the emails to 

`POST http://countryconfig:3040/email`
```
{
  subject: '{{HOST}} alert: Error in service opencrvs_metrics 🚨',
  html: 'Login to https://kibana.{{HOST}}/app/apm/services/opencrvs_metrics/overview to view the alert.',
  from: '{{SENDER_EMAIL_ADDRESS}}',
  to: '{{ALERT_EMAIL}}'
}
```

The generic email endpoint then has to replace the moustaches with values that are in country config's environment. This is also to reduce the amount of environment variables we need to pass around as part of deployments.

### CODE REVIEW

File: infrastructure/monitoring/elastalert/rules/alert.yaml [hunk 1]
```diff
@@ -42,12 +35,10 @@ filter:
...
-smtp_port: {{SMTP_PORT}}
-smtp_ssl: false
-smtp_auth_file: '/opt/elastalert/auth.yaml'
+alert: post2
+http_post2_url: 'http://countryconfig:3040/email'
+http_post2_payload:
+  subject: '{% raw %}{{DOMAIN}}{% endraw %} {{kibana.alert.context.metrics__alert__inventory__threshold.alertState}}: {{rule.name}} 🚨'
+  html: 'Reason: {{kibana.alert.context.metrics__alert__inventory__threshold.reason}}. Login to https://kibana.{% raw %}{{DOMAIN}}{% endraw %} to view the alert.'
+  from: '{% raw %}{{SENDER_EMAIL_ADDRESS}}{% endraw %}'
+  to: '{% raw %}{{ALERT_EMAIL}}{% endraw %}'
```
Comment (Zangetsu101) [inline infrastructure/monitoring/elastalert/rules/alert.yaml:44]: So all the raw ones will get replaced in country-config? Nice one

File: src/api/notification/constant.ts [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+
 /* SMTP (Email) */
 export const SMTP_HOST = process.env.SMTP_HOST
 export const SMTP_PORT = process.env.SMTP_PORT
-export const SMTP_USER = process.env.SMTP_USER
-export const SMTP_PASS = process.env.SMTP_PASS
+  ? parseInt(process.env.SMTP_PORT, 10)
+  : 587
+export const SMTP_USERNAME = process.env.SMTP_USERNAME
+export const SMTP_PASSWORD = process.env.SMTP_PASSWORD
```
Comment (Zangetsu101) [inline src/api/notification/constant.ts:15]: we will need to save the appropriate values for these two in our environments

**Review by Zangetsu101 — COMMENTED** [review]
Looks great! Awesome work Riku

**Review by Zangetsu101 — APPROVED** [review]

File: infrastructure/monitoring/elastalert/rules/alert.yaml (hunk 1 above)
Comment (rikukissa) [reply infrastructure/monitoring/elastalert/rules/alert.yaml:44]: Exactly 👍 Removes the need for deploy-time search and replace of these files

**Pull request merged**